features/internationalization.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Internationalization
+  The CCA DIP access application will have an interface that can be shown in
+  different languages, being English the source language. On this first iteration
```

Comment:
change 'being English the source language' to

'with English as the default language'